routes/index.js

```diff
@@ -0,0 +1,68 @@
+const express = require('express');
+const router = express.Router();
+const Twit = require('twit');
+const config = require('./config.js')
+const { Twitter } = require('../database/database.js')
+
+router.get('/', function(req, res, next) {
+  Twitter.getAllTweets().then((results) => {
+    res.render('index', {results})
+  })
+})
+
+console.log('this bot is running...');
+
+let T = new Twit(config);
+
+let techCrunchSearch = {
```

Comment:
This seems like it should be a `const`. You aren't wanting and/or expecting this object to mutate later so I would make it immutable with `const`.

PS. Rule of thumb for const vs let: 
```
Always use const unless you can't
```